### PR TITLE
Combine sorts, plus small improvement.

### PR DESCRIPTION
**Pull request by whiteroses**
Combine sorts as suggested by @mmerickel [here](https://github.com/Pylons/webob/pull/399#discussion_r289913511), plus small improvement by removing an unnecessary assignment.

Thanks!

### CODE REVIEW

File: src/webob/acceptparse.py [hunk 1]
```diff
@@ -1003,17 +1003,16 @@ def acceptable_offers(self, offers):
...
+            reverse=True,
+            # descending sort by (qvalue, -offer_index)
         )
-        # drop offer_index
-        acceptable_offers_n_quality_factors = [
-            (item[0], item[1]) for item in acceptable_offers_n_quality_factors
+        return [
+            (item[0], item[1])
+            for item in acceptable_offers_n_quality_factors
+            # dropping offer_index
```
Comment (bertjwregeer) [inline src/webob/acceptparse.py]: Don't put this inside the list comprehension, it currently causes a bug in black to be exposed which may cause issues when formatting the code base.

Just place it on the line before the `return` please.

Comment (whiteroses) [reply src/webob/acceptparse.py]: Do you mean the comment @bertjwregeer?

Comment (bertjwregeer) [reply src/webob/acceptparse.py]: Yes, I mean the comment. See https://github.com/python/black/issues/563 for the issue.

Comment (whiteroses) [reply src/webob/acceptparse.py]: Ok, I've made the change.

**Comment by bertjwregeer** [comment]
Thanks @whiteroses!

**Pull request merged**